app/Http/Resources/Models/LanguageLineResource.php

```diff
@@ -19,13 +21,13 @@ class LanguageLineResource extends BaseResource
      *
      * @param \Illuminate\Http\Request $request
      *
-     * @return array<string, string>
+     * @return array<string, mixed>
      */
-    public function toArray($request): array
+    public function toArray(Request $request): array
     {
         return [
             'key'  => $this->resource->key,
-            'text' => strval(__($this->resource->handle)),
+            'text' => $this->resource->getTranslation(App::getLocale()),
```

Comment:
I find it counterintuitive to use `handle` here, when we can easily use `text` attribute -- which is the attribute being parsed.